how-we-work/oncall-responsibilities.md

```diff
@@ -20,13 +20,13 @@ An on-call rotation begins on Monday at 9am PT. Each week, there is a primary an
 
 ##### Deployment summary table
 
```

Comment:
Could you add a sentence here describing the below table please? It took me a second to figure out what exactly the table was summarizing.

Reply:
done! clarified content as well (hopefully!)